src/components/InventoryList.tsx

```diff
@@ -36,6 +47,17 @@ const InventoryList: React.FC = () => {
     history.push(path);
   };
 
+  // handler for changing sort direction
+  const onSortDirectionChange = () => {
```

Comment:
I don't understand why this works. I cannot find the place the `sortDirection` actually changes with the click?

Reply:
By clicking, you circle through the different states. But I don't find the place where this is initiated.